portfolio/src/main/java/com/google/sps/data/Login.java

```diff
@@ -0,0 +1,23 @@
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** An item on a todo list. */
+public final class Login {
+
+  private final boolean status;
```

Comment:
I don't know the answer: does it work with these fields being private?

Reply:
It does. Same with the comments